### DISCUSSION TITLE
Scrypted NVR Hardware requirements

### DISCUSSION THREAD

**Discussion by Dishboy007** [koush/scrypted#1217] [Q&A] [answered]
Would a Dell Optiplex 7050 Core i7-7700 3.6ghz 32GB with a CORAL TPU DUAL EDGE TPU and maybe a GPU running on Ubuntu be enough to run a 12 camera Scrypted NVR with object/motion detection? I posted on reddit page, but had zero replies.

**Comment by koush** [comment, marked as answer]
Yes more than enough. An N100 would probably run just as well for cheaper and less power too.

**Reply by Dishboy007** [reply]
Thanks!
